cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -1390,10 +1390,10 @@ def _validate_spatial_tissue_position(self, tissue_position_name: str, min: int,
         :rtype none
         """
         # Tissue position is foribidden if assay is not Visium and is_single is True.
-        if (
-            tissue_position_name in self.adata.obs
-            and (
-                ~((self.adata.obs["assay_ontology_term_id"] == ASSAY_VISIUM) & (self.adata.uns["spatial"]["is_single"]))
+        if tissue_position_name in self.adata.obs and (
+            not self._is_supported_spatial_assay
+            or (
+                ~((self.adata.obs["assay_ontology_term_id"] == ASSAY_VISIUM) & ((self._is_single() or False)))
                 & (self.adata.obs[tissue_position_name].notnull())
             ).any()
         ):
```

Comment:
I think we can simplify this by doing the following--would this work and still cover everything?

```suggestion
        if tissue_position_name in self.adata.obs and (
            not self._is_visium_and_is_single_true()
            or (
                ~(self.adata.obs["assay_ontology_term_id"] == ASSAY_VISIUM)
                & (self.adata.obs[tissue_position_name].notnull())
            ).any()
        ):
```